File: add/codesnippet/CSharp/m-system.web.services.de_0_1.cs
            // Get another message from ServiceDescription.
            Message myMessage2 = myServiceDescription.Messages["DivideHttpGetOut"];
            MessagePart myMessagePart=myMessage2.FindPartByName("Body");
            Console.WriteLine("Results of FindPartByName operation:");
            Console.WriteLine("Part Name: " +myMessagePart.Name);
            Console.WriteLine("Part Element: " +myMessagePart.Element);

File: add/codesnippet/CPP/m-system.web.services.de_0_1.cpp
      // Get another message from ServiceDescription.
      Message^ myMessage2 = myServiceDescription->Messages[ "DivideHttpGetOut" ];
      MessagePart^ myMessagePart = myMessage2->FindPartByName( "Body" );
      Console::WriteLine( "Results of FindPartByName operation:" );
      Console::WriteLine( "Part Name: {0}", myMessagePart->Name );
      Console::WriteLine( "Part Element: {0}", myMessagePart->Element );